definitions/infra-awsapigatewayresourcewithmetrics/golden_metrics.yml

```diff
@@ -0,0 +1,24 @@
+requests:
+  title: Requests
+  query:
+    select: sum(provider.count.SampleCount)
+    from: ApiGatewaySample
+    where: provider='ApiGatewayResourceWithMetrics'
+    facet: entityName
+    eventId: entityGuid
+errors4XxAnd5Xx:
```

Comment:
nit: `4xx` or `4XX` same with `5Xx` this key is used to generate pre computed metrics so we should keep them standard.